xml/System.Numerics/BigInteger.xml

```diff
@@ -1482,11 +1482,15 @@
 ## Remarks  
  This property is a convenience feature that indicates whether a <xref:System.Numerics.BigInteger> value is evenly divisible by two. It is equivalent to the following expression:  
   
-```csharp  
+ [C#]  
```

Comment:
these changes showing up here is telling me you have an old version of our content. do you know how to rebase against the latest master? if not, I can send you instructions offline.